node-feature-discovery-0.15.yaml

```diff
@@ -0,0 +1,94 @@
+package:
+  name: node-feature-discovery-0.15
+  version: 0.15.4
+  epoch: 0
+  description: Node feature discovery for Kubernetes
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - protoc
+    provides:
+      - node-feature-discovery=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - protoc
+  environment:
+    GRPC_GO_LOG_SEVERITY_LEVEL: "INFO"
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/kubernetes-sigs/node-feature-discovery
+      tag: v${{package.version}}
+      expected-commit: 08c731f67fddff47b13aad83fafc2d4e745a72cd
+
+  - name: copy conf
+    runs: |
+      mkdir -p ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/
+      cp ./deployment/components/worker-config/nfd-worker.conf.example ${{targets.contextdir}}/etc/kubernetes/node-feature-discovery/nfd-worker.conf
+
+  - uses: go/build
```

Comment:
The image just throws all of these into /usr/bin so we should be good without splitting these up into subpackages